### PR TITLE
add ETMP network

### PR DESCRIPTION
**Pull request by LuxesOwen**
chainID is already added in chainlist :
eip155-48.json: https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-48.json
eip155-49.json: https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-49.json

### CODE REVIEW

**Review by moisses89 — CHANGES_REQUESTED** [review]
I couldn't find a valid RPC https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-48.json for `Ennothem Mainnet Proterozoic` Then I can't validate the addresses.
For testnet also doesn't seems the deterministic addresses, ensure that you are in the correct branch https://github.com/safe-global/safe-contracts#deploy

**Comment by LuxesOwen** [comment]
we add rpc address in mainnet , and updated ETMP testnet contract address, please check again

File: src/assets/v1.3.0/compatibility_fallback_handler.json [hunk 1]
```diff
@@ -16,6 +16,8 @@
     "41": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "42": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "5": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "48": "0x0D2068Bbe4e3975adc8D6701234fBdA115CdAc19",
+    "49": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
```
Comment (moisses89) [inline src/assets/v1.3.0/compatibility_fallback_handler.json:20]: Did you deploy the contracts on testnet?
```curl --location --request POST 'https://rpc.pioneer.etm.network' \
--header 'Content-Type: application/json' \
--data-raw '{
    "jsonrpc": "2.0",
    "method": "eth_getCode",
    "params": [
        "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
        "latest"
    ],
    "id": 1}'
```
If I ask for the code to the RPC returns empty
```
{"jsonrpc":"2.0","id":1,"result":"0x"}% 
```

File: src/assets/v1.3.0/compatibility_fallback_handler.json [hunk 2]
```diff
@@ -16,6 +16,8 @@
     "41": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "42": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "5": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "48": "0x0D2068Bbe4e3975adc8D6701234fBdA115CdAc19",
```
Comment (moisses89) [inline src/assets/v1.3.0/compatibility_fallback_handler.json:19]: The contracts should be deployed also in mainnet using the deterministic addresses.

**Review by moisses89 — CHANGES_REQUESTED** [review]

**Comment by Uxio0** [comment]
I'm closing this issue due to inactivity

**Pull request closed without merging**